src/etc/inc/system.inc

```diff
@@ -73,7 +73,9 @@ function system_sysctl_defaults()
         'debug.kassert.warn_only' => [ 'default' => '1', 'required' => true, 'description' => 'KASSERT triggers a panic (0) or just a warning (1)', 'type' => 'w' ],
         'hw.ibrs_disable' => [ 'default' => '0' ],
         'hw.ixl.enable_head_writeback' => [ 'default' => '0', 'required' => true ],
+        'hw.vtnet.csum_disable' => [ 'default' => '1', 'required' => true ],
```

Comment:
```suggestion
```

Reply:
If I commit both your suggested changes, what's left? What is your intention?

Reply:
1 single line that is needed for this to work, see my first comment.

Reply:
But that is already the case: https://github.com/opnsense/core/pull/8007/files

Your new suggested change removes even that one so nothing is left.

Reply:
Look at the diff in GitHub. It was added twice.

Reply:
Done :) My bad.